<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cool Analyser</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>文件类型</string>
   </property>
   <widget class="QRadioButton" name="rbMMS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>彩信</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNewspaper">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>手机报</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>选择文件和筛选器</string>
   </property>
   <widget class="QPushButton" name="btnOpenDialog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFilterField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFilterValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbFilterMap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>74</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>按组</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="lbFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddFilter">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>30</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="lbProcessor">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>统计</string>
   </property>
   <widget class="QComboBox" name="cbStatField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStatValue">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbStatMap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>按组</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddStat">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStatRoutine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>报告</string>
   </property>
   <widget class="QRadioButton" name="rdTxt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>txt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdXls">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>xls</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>display</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edSaveFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>out.xls</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>501</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>执行记录</string>
    </attribute>
    <widget class="QTextBrowser" name="edtStatus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
